api/Document.json

```diff
@@ -1932,7 +1932,7 @@
               "version_added": false
             },
             "ie": {
-              "version_added": null
+              "version_added": false
```

Comment:
`document.createEntityReference` confirmed absent in IE11 by https://software.hixie.ch/utilities/js/live-dom-viewer/saved/8472